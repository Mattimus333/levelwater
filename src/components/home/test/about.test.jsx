import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import About from '../about.jsx';

describe('about', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
  });
  it('renders a snapshot', () => {
    const tree = shallow(<About />);
    expect(tree).toMatchSnapshot();
  });
});
