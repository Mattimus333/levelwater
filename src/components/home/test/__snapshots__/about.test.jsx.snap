exports[`about renders a snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="about-container">
    <div
        className="about-heading">
        <h3>
            About America\'s Water Infrastructure
        </h3>
    </div>
    <div
        className="about">
        <div
            className="ui grid">
            <div
                className="sixteen wide column">
                <p>
                    This year the American Society of Civil Engineers evaluated the USA\'s drinking water infrastructure and rated it a \'D\'. There are 155,000 public water systems in the country, the majority of which are smaller systems that lack the economies of scale and the technical, manegerial, and financial capacity that large systems have. This lack of asset management means that many are not meeting state requirements or those of the Safe Drinking Water Act, and may be one capital malfunction away from going bankrupt and leaving their customers dry.
                </p>
            </div>
        </div>
    </div>
    <div
        className="about-heading">
        <h3>
            About levelwater.io
        </h3>
    </div>
    <div
        className="about">
        <div
            className="ui grid">
            <div
                className="sixteen wide column">
                <p>
                    At levelwater.io we believe that America\'s drinking water deserves an \'A\'. We strive to help small water systems improve their infrastructure by providing a user-friendly, low-overhead asset management system to help ensure that they are keeping track of all of their capital assets and when they need to be replaced, as well as using a complex algorithm to let them know how they should be setting their rates currently as well as in the years to come.
                </p>
            </div>
        </div>
    </div>
</div>,
  "nodes": Array [
    <div
      className="about-container">
      <div
            className="about-heading">
            <h3>
                  About America\'s Water Infrastructure
            </h3>
      </div>
      <div
            className="about">
            <div
                  className="ui grid">
                  <div
                        className="sixteen wide column">
                        <p>
                              This year the American Society of Civil Engineers evaluated the USA\'s drinking water infrastructure and rated it a \'D\'. There are 155,000 public water systems in the country, the majority of which are smaller systems that lack the economies of scale and the technical, manegerial, and financial capacity that large systems have. This lack of asset management means that many are not meeting state requirements or those of the Safe Drinking Water Act, and may be one capital malfunction away from going bankrupt and leaving their customers dry.
                        </p>
                  </div>
            </div>
      </div>
      <div
            className="about-heading">
            <h3>
                  About levelwater.io
            </h3>
      </div>
      <div
            className="about">
            <div
                  className="ui grid">
                  <div
                        className="sixteen wide column">
                        <p>
                              At levelwater.io we believe that America\'s drinking water deserves an \'A\'. We strive to help small water systems improve their infrastructure by providing a user-friendly, low-overhead asset management system to help ensure that they are keeping track of all of their capital assets and when they need to be replaced, as well as using a complex algorithm to let them know how they should be setting their rates currently as well as in the years to come.
                        </p>
                  </div>
            </div>
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <About />,
      "_debugID": 15,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="about-container">
          <div
                    className="about-heading">
                    <h3>
                              About America\'s Water Infrastructure
                    </h3>
          </div>
          <div
                    className="about">
                    <div
                              className="ui grid">
                              <div
                                        className="sixteen wide column">
                                        <p>
                                                  This year the American Society of Civil Engineers evaluated the USA\'s drinking water infrastructure and rated it a \'D\'. There are 155,000 public water systems in the country, the majority of which are smaller systems that lack the economies of scale and the technical, manegerial, and financial capacity that large systems have. This lack of asset management means that many are not meeting state requirements or those of the Safe Drinking Water Act, and may be one capital malfunction away from going bankrupt and leaving their customers dry.
                                        </p>
                              </div>
                    </div>
          </div>
          <div
                    className="about-heading">
                    <h3>
                              About levelwater.io
                    </h3>
          </div>
          <div
                    className="about">
                    <div
                              className="ui grid">
                              <div
                                        className="sixteen wide column">
                                        <p>
                                                  At levelwater.io we believe that America\'s drinking water deserves an \'A\'. We strive to help small water systems improve their infrastructure by providing a user-friendly, low-overhead asset management system to help ensure that they are keeping track of all of their capital assets and when they need to be replaced, as well as using a complex algorithm to let them know how they should be setting their rates currently as well as in the years to come.
                                        </p>
                              </div>
                    </div>
          </div>
</div>,
        "_debugID": 16,
        "_renderedOutput": <div
          className="about-container">
          <div
                    className="about-heading">
                    <h3>
                              About America\'s Water Infrastructure
                    </h3>
          </div>
          <div
                    className="about">
                    <div
                              className="ui grid">
                              <div
                                        className="sixteen wide column">
                                        <p>
                                                  This year the American Society of Civil Engineers evaluated the USA\'s drinking water infrastructure and rated it a \'D\'. There are 155,000 public water systems in the country, the majority of which are smaller systems that lack the economies of scale and the technical, manegerial, and financial capacity that large systems have. This lack of asset management means that many are not meeting state requirements or those of the Safe Drinking Water Act, and may be one capital malfunction away from going bankrupt and leaving their customers dry.
                                        </p>
                              </div>
                    </div>
          </div>
          <div
                    className="about-heading">
                    <h3>
                              About levelwater.io
                    </h3>
          </div>
          <div
                    className="about">
                    <div
                              className="ui grid">
                              <div
                                        className="sixteen wide column">
                                        <p>
                                                  At levelwater.io we believe that America\'s drinking water deserves an \'A\'. We strive to help small water systems improve their infrastructure by providing a user-friendly, low-overhead asset management system to help ensure that they are keeping track of all of their capital assets and when they need to be replaced, as well as using a complex algorithm to let them know how they should be setting their rates currently as well as in the years to come.
                                        </p>
                              </div>
                    </div>
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <About />,
}
`;
